src/test/java/com/kainos/ea/backend/controllers/UserControllerTest.java

```diff
@@ -0,0 +1,123 @@
+package com.kainos.ea.backend.controllers;
+
+import com.kainos.ea.backend.models.User;
+import com.kainos.ea.backend.services.UserService;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(MockitoExtension.class)
+public class UserControllerTest {
+
+    @Mock
+    private UserService userService;
+
+    @Mock
+    private BCryptPasswordEncoder passwordEncoder;
+
+    @Test
+    public void when_verifyUser_called_with_valid_user_expect_true() {
+        User validUser = new User("mail@email.com", "strong_password");
+        List<User> users = List.of(new User("mail@email.com", "strong_password"));
```

Comment:
I assume there is a reason you can't use the `validUser` object here?

Reply:
Changed to use validUser